server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -4301,9 +4388,13 @@ public void getApplicationEligibilityStatus() {
     assertThat(
             subject.getApplicationEligibilityStatus(ineligibleApplication, programDefinition).get())
         .isFalse();
+    assertThat(ineligibleApplication.getEligibilityDetermination())
+        .isEqualTo(EligibilityDetermination.INELIGIBLE);
     // Re-submission evaluates to eligible.
     assertThat(
             subject.getApplicationEligibilityStatus(eligibleApplication, programDefinition).get())
         .isTrue();
+    assertThat(eligibleApplication.getEligibilityDetermination())
+        .isEqualTo(EligibilityDetermination.ELIGIBLE);
```

Comment:
Thanks for adding these asserts! Would you add one more to a test that submits an application to a program with no eligibility settings to make sure it correctly sets it to `EligibilityDetermination.NO_ELIGIBILITY_CRITERIA`?